static/usage/v7/toast/swipe-gesture/vue.md

```diff
@@ -0,0 +1,35 @@
+```html
+<template>
+  <ion-content class="ion-padding">
+    <ion-button id="open-toast">Open Toast</ion-button>
+    <ion-toast
+      message="This toast can be swiped to dismiss"
+      trigger="open-toast"
+      swipe-gesture="vertical"
+      position="bottom"
+      position-anchor="footer"
+    ></ion-toast>
+  </ion-content>
+  <ion-footer id="footer">
+    <ion-toolbar>
+      <ion-title>Footer</ion-title>
+    </ion-toolbar>
+  </ion-footer>
+</template>
+
+<script lang="ts">
```

Comment:
Examples look good, should we be using `<script setup>` on new playground examples so we don't need to migrate them later?

e.g.:
```
<template>
  <ion-content class="ion-padding">
    <ion-button id="open-toast">Open Toast</ion-button>
    <ion-toast
      message="This toast can be swiped to dismiss"
      trigger="open-toast"
      swipe-gesture="vertical"
      position="bottom"
      position-anchor="footer"
    ></ion-toast>
  </ion-content>
  <ion-footer id="footer">
    <ion-toolbar>
      <ion-title>Footer</ion-title>
    </ion-toolbar>
  </ion-footer>
</template>

<script setup lang="ts">
import {
  IonButton,
  IonContent,
  IonFooter,
  IonTitle,
  IonToast,
  IonToolbar,
} from '@ionic/vue';
</script>
```

https://stackblitz.com/edit/8hheka?file=src%2Fcomponents%2FExample.vue

Cuts down on a little of the boilerplate required.

Reply:
Done! https://github.com/ionic-team/ionic-docs/pull/3247/commits/cce5dc81420162d9adbbc6a8d37d2a2a58f59315